pages/windows/taskkill.md

```diff
@@ -0,0 +1,27 @@
+# taskkill
+
+> Terminate a process by its process id or name.
+
+- Display information about the usage of the command:
```

Comment:
Move this to the end.

TIP: Arrange the examples in the order you expect the users to use them. Usually we keep the `-h` and its variants at the end, as those are expected to let the user know about further usages of the command.